app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -19,6 +19,15 @@ class GetUspsProofingResultsJob < ApplicationJob
 
   discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
+  def email_analytics_attributes(enrollment, delay_time)
+    {
+      timestamp: Time.zone.now,
+      user_id: enrollment.user_id,
+      service_provider: enrollment.issuer,
+      delay_time_seconds: delay_time,
```

Comment:
Since the delivery parameter method is in-scope from this method, what would you think about calling it directly rather than expecting it to be passed as a parameter, optionally refactoring out a method to just get the delay value?

```suggestion
  def email_analytics_attributes(enrollment)
    {
      timestamp: Time.zone.now,
      user_id: enrollment.user_id,
      service_provider: enrollment.issuer,
      delay_time_seconds: mail_delivery_delay_hours.in_seconds,
```